File: main/dto/SoldTicketDto.cs
using System.Text.Json.Serialization;

namespace main.dto
{
    public class SoldTicketDto
    {
        [JsonPropertyName("id")]
        public long? Id { get; set; }
        [JsonPropertyName("soldBy")]
        public UserDto SoldBy { get; set; }
        [JsonPropertyName("game")]
        public GameDto Game { get; set; }
        [JsonPropertyName("soldAt")]
        public DateTime SoldAt { get; set; }
        [JsonPropertyName("noOfTickets")]
        public int NumberOfSeats { get; set; }
        [JsonPropertyName("username")]
        public string Username { get; set; }
    }
}

File: java-client/src/main/java/cs/ubb/mpp/client/dto/SoldTicketDto.java
package cs.ubb.mpp.client.dto;

import java.time.LocalDateTime;

public class SoldTicketDto {
    private Long id;
    private UserDto soldBy;
    private GameDto game;
    private LocalDateTime soldAt;
    private int noOfTickets;
    private String username;

    public SoldTicketDto(Long id, UserDto soldBy, GameDto game, LocalDateTime soldAt, int noOfTickets, String username) {
        this.id = id;
        this.soldBy = soldBy;
        this.game = game;
        this.soldAt = soldAt;
        this.noOfTickets = noOfTickets;
        this.username = username;
    }

    public UserDto getSoldBy() {
        return soldBy;
    }

    public GameDto getGame() {
        return game;
    }

    public LocalDateTime getSoldAt() {
        return soldAt;
    }

    public int getNoOfTickets() {
        return noOfTickets;
    }

    public String getUsername() {
        return username;
    }

    public Long getId() {
        return id;
    }

    @Override
    public String toString() {
        return "SoldTicketDto{" +
                "soldBy=" + soldBy +
                ", game=" + game +
                ", soldAt=" + soldAt +
                ", noOfTickets=" + noOfTickets +
                ", username='" + username + '\'' +
                '}';
    }
}
